packages/server/src/tests/header/headerSelection.ts

```diff
@@ -27,35 +27,45 @@ const nonSupportersTestNonUK: HeaderTest = {
     ],
 };
 
-const nonSupportersTestUS: HeaderTest = {
-    name: 'RemoteRrHeaderLinksTest__USEOY',
-    audience: 'AllNonSupporters',
-    variants: [
-        {
-            name: 'remote',
-            modulePathBuilder: usEOYHeader.endpointPathBuilder,
-            content: {
-                heading: 'Support the Guardian',
-                subheading: 'Make a year-end gift',
-                primaryCta: {
-                    url: 'https://support.theguardian.com/contribute',
-                    text: 'Contribute',
-                },
-                secondaryCta: {
-                    url: 'https://support.theguardian.com/subscribe',
-                    text: 'Subscribe',
+const nonSupportersTestUS = (): HeaderTest => {
+    const givingTuesdayStart = new Date('2021-11-29T17:00:00'); //remove "Subscribe" Monday 12:00 PM EST
+    const givingTuesdayEnd = new Date('2021-11-01T09:00:00'); //re-add "Subscribe" on Wednesday morning GMT
```

Comment:
`givingTuesdayEnd` is 1st of December: `const givingTuesdayEnd = new Date('2021-12-01T09:00:00');`

Reply:
Hi @GHaberis, as the date library is indexed from 0 we had to reduce the month by 1.

Reply:
Oh good spot George! @samanthagottlieb we'd need to reduce the month by 1 if we used

```ts
new Date(2021, 11, 01) // 1st of dec
```

but not if we use a string

```ts
new Date('2021-12-01') // 1st of dec
```

Reply:
Thanks both - yup, we were initially using the date-fns library, hence the confusion! @marialani and I are just creating a PR to fix this now 🙂

Reply:
Yeah we were initially using the date-fns library but the 0 based indexing for months was a bit confusing so this was updated to use the Date object but some of the logic had been muddled in translation. This has since been updated on [this PR](Update showing subscribe button in header logic by marialani · Pull Request #585 · guardian/support-dotcom-components
Update showing subscribe button in header logic by marialani · Pull Request #585 · guardian/support-dotcom-components
github.com)